### ISSUE TITLE
Grab Document title from public page

### ISSUE THREAD

**Issue by iversond** [psadmin-io/mosbot-cli#13]
For MOS documents that have a public page, grab the title and display it with the URL.

Example, `mos doc 2306268.1` has this public page:

https://support.oracle.com/knowledge/PeopleSoft%20Enterprise/2306268_1.html

**Comment by iversond** [comment]
Some of the titles have quotes and may not render properly. E.g,

```
$ mosbot doc 983986.1
E-PORTAL: &quot;Could not open registry&quot; Error with Single Link SSO from Portal to Content.
https://support.oracle.com/epmos/faces/DocumentDisplay?id=983986.1
```

**Comment by bmcculley** [comment]
@iversond oops, I didn't consider HTML encoding being in the title. Just opened #17 for this, I used the cgi library to [unescape any HTML](https://ruby-doc.org/stdlib-2.3.1/libdoc/cgi/rdoc/CGI/Util.html#method-i-unescapeHTML) encoding in the title.

**Comment by iversond** [comment]
:) No worries. I merged #17 and will push out a new gem build (v2.0.1) to RubyGems.
